README.md

```diff
@@ -92,14 +99,19 @@ Example:
   });
 
   const untrusted_example = "Well, <em class=nonchalant onclick='alert(\'General Kenobi\');'><a href='https://obiwan.org/home.php'>hello there<a>!"
```

Comment:
obiwan.org seems to be an existing site? Can we use obiwan.example?

Reply:
Done.

(I can't find a page or whois entry to obiwan.org, but let's use the reserved .example to be safe.)